util/winutil/winutil_windows.go

```diff
@@ -18,15 +22,21 @@ const (
 )
 
 // GetDesktopPID searches the PID of the process that's running the
-// currently active desktop and whether it was found.
+// currently active desktop or error.
 // Usually the PID will be for explorer.exe.
-func GetDesktopPID() (pid uint32, ok bool) {
+func GetDesktopPID() (pid uint32, err error) {
 	hwnd := windows.GetShellWindow()
 	if hwnd == 0 {
-		return 0, false
+		err = errors.New("no Shell process is present")
```

Comment:
add global `var ErrNoShell = ...` and then:

return 0, ErrNoShell

... so this doesn't allocate in the two common cases?

Then you can document it returns ErrNoShell when it's missing.

But OTOH what code cares about the distinction of ok bool vs error anyway?